src/elli_prometheus_config.erl

```diff
@@ -0,0 +1,32 @@
+-module(elli_prometheus_config).
+
+-export([path/0,
+         format/0,
+         duration_buckets/0]).
+
+-define(DEFAULT_PATH, <<"/metrics">>).
+-define(DEFAULT_FORMAT, prometheus_text_format).
+-define(DEFAULT_DURATION_BUCKETS, [10, 100, 1000, 10000, 100000, 300000, 500000,
```

Comment:
Is it possible/wise to use one of the bucket-creating functions here instead of hardcoding?